Formula/zeromq.rb

```diff
@@ -11,47 +12,56 @@ class Zeromq < Formula
     sha256 "78c24719c7c8dbba157ca225b41d515190281eb27549caaa60a359ec9f33e3f1" => :yosemite
   end
 
-  head do
-    url "https://github.com/zeromq/libzmq.git"
-
-    depends_on "autoconf" => :build
-    depends_on "automake" => :build
-    depends_on "libtool" => :build
-  end
-
   option :universal
   option "with-libpgm", "Build with PGM extension"
   option "with-norm", "Build with NORM extension"
+  option "with-drafts", "Build and install draft classes and methods"
```

Comment:
Can you explain a bit more about what this does and why you'd want to enable/disable it?